app/tests/spec/views/mixins/password-prompt-mixin.js

```diff
@@ -0,0 +1,132 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define(function (require, exports, module) {
+  'use strict';
+
+  var $ = require('jquery');
+  var chai = require('chai');
+  var Cocktail = require('cocktail');
+  var PasswordPromptMixin = require('views/mixins/password-prompt-mixin');
+  var PasswordStrengthMixin = require('views/mixins/password-strength-mixin');
+  var sinon = require('sinon');
+
+  var FormView = require('views/form');
+  var Template = require('stache!templates/test_template');
+
+  var TestView = FormView.extend({
+    template: Template
+  });
+
+  Cocktail.mixin(
+    TestView,
+    PasswordPromptMixin,
+    PasswordStrengthMixin
+  );
+
+  var TOOLTIP_MESSAGES = {
+    FOCUS_PROMPT_MESSAGE: '8 characters minimum, but longer if you plan to sync passwords.',
+    INITIAL_PROMPT_MESSAGE: 'A strong, unique password will keep your Firefox data safe from intruders.',
+    WARNING_PROMPT_MESSAGE: 'This is a common password; please consider another one.'
+  };
+  var PASSWORD_FIELD_SELECTOR = '.check-password';
+  var TOOLTIP_SELECTOR = '.tooltip-warning';
```

Comment:
nit: it would be cool for `password-prompt-mixin.js` to export the messages in some way so we don't have to repeat these 3 variables above.